### PR TITLE
Update view-directory-synchronization-status.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated navigation

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8681

File: microsoft-365/enterprise/view-directory-synchronization-status.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ If you have integrated your on-premises Active Directory Domain Services (AD DS)
 ## View directory synchronization status
 
 - Sign in to the [Microsoft 365 admin center](https://admin.microsoft.com) and choose **DirSync Status** on the home page.
-- Alternately, you can go to **Users** \> **Active users**, and on the **Active users** page, choose **More** \> **Directory synchronization**. On the **Directory Synchronization** pane, choose **Go to DirSync management**.
+- Alternately, you can go to **Users** \> **Active users**, and on the **Active users** page, Click on the **Elipse** \> **Directory synchronization**. On the **Directory Synchronization** pane, choose **Go to DirSync management**.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/view-directory-synchronization-status.md]:
```suggestion
- Alternately, you can go to **Users** \> **Active users**, and on the **Active users** page, select the **Elipse** \> **Directory synchronization**. On the **Directory Synchronization** pane, choose **Go to DirSync management**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**